youths/schema.py

```diff
@@ -177,6 +178,11 @@ class Input:
     def mutate_and_get_payload(cls, root, info, **input):
         input_data = input.get("youth_profile")
 
+        if calculate_age(input_data["birth_date"]) < 13:
```

Comment:
is it okay that the user will be able to modify the birth date later on thru `UpdateMyYouthProfileMutation` and thus her age to below 13 y.o.?

Reply:
According to current spec, yes 🙄

Reply:
well then 👍